src/main/java/com/devcomanda/easymeetup/controller/MeetupController.java

```diff
@@ -37,4 +39,20 @@ public MeetupController(final MeetupService meetUpService) {
         }
         return new ResponseEntity<>(meetups, HttpStatus.FOUND);
     }
+
+    @PutMapping(path = "/meetups/{id}", produces = "application/json;charset=UTF-8")
+    public ResponseEntity<Meetup> updateMeetup(@PathVariable ("id") Long id, @Valid @RequestBody Meetup meetup){
+        Meetup foundMeetup = meetUpService.findMeetuById(id);
+        if (foundMeetup == null){
+            return new ResponseEntity<>(HttpStatus.NO_CONTENT);
+        }
+        foundMeetup.setMeetupName(meetup.getMeetupName());
```

Comment:
You should move the logic with updating meetup to service,
That needs for using transactional on service level and possibility reuse that code in another services